client/fabfile.py

```diff
@@ -107,7 +107,7 @@ def package(package_requirements='yes', flavor=''):
         package_python_openbts(package_requirements)
     elif (env.gsmeng == "osmocom"):
         package_python_osmocom(package_requirements)
-        package_freeswitch_mod_smpp()
+        #package_freeswitch_mod_smpp()
```

Comment:
It should be fine to remove from the package command. Most people will not need to rebuild freeswitch in their development flow